package.yaml

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: 2020 TBD
+# SPDX-License-Identifier: LicenseRef-Proprietary
+<<: !include "./hpack/module.yaml"
+
+name:                globacap
+version:             0.1.0
+
+extra-source-files:
+- README.md
+
+category:            Blockchain
+description:
+  FA1.2-based token contract (called `Holdings`) with
+  `Safelist` contract that contains information about whitelisted
+  and blacklisted investors.
+
+library:
+  <<: *lib-common
+
+  default-extensions:
+    - DerivingStrategies
+    - DeriveAnyClass
+    - NoApplicativeDo
```

Comment:
Endless useless. `ApplicativeDo` is disabled by default, there's no need to disable it additionally. And why are these default extensions not in the other list of default extensions (in `/hpack`)?